src/HumanView.cpp

```diff
@@ -125,15 +129,13 @@ void HumanView::drawDialogueBox() {
 }
 
 void HumanView::drawLevelDialogue() {
-    if (first) {
```

Comment:
Why is the if statement not needed here?

Reply:
Initially I had the first variable only checking for certain things in the draw methods, but I changed it so that each draw method is only called once from update. Basically, I moved the if statement from the draw method up to the update method and I had not actually updated my code since I did that. The if statement inside these draw methods doesn't matter because of the if in update.